package.json

```diff
@@ -28,6 +28,7 @@
   },
   "homepage": "https://github.com/brentvatne/react-native-scrollable-tab-view#readme",
   "dependencies": {
+    "react-static-container": "^1.0.1",
```

Comment:
https://github.com/facebook/react-native/blob/50c2467905815dbb434beecd0e649fb4a7bc2e20/Libraries/react-native/react-native.js
https://github.com/facebook/react-native/blob/50c2467905815dbb434beecd0e649fb4a7bc2e20/Libraries/Components/StaticContainer.js
Though it is shipped in the code, its not exported to the main ReactNative object.
It's not working like in the tabbarios, so had to do it from external component(i think its an offical release , not sure)
@skv-headless Please suggest alternative if not that plugin.I would like to wrap this whole thing today itself